test_requirements.txt

```diff
@@ -2,4 +2,5 @@ coverage==3.7
 nose==1.3.0
 mock==1.0.1
 pep8==1.7.0
+requests==2.10.0
```

Comment:
Previously it got installed "for free" as a subdependency of browsermob-proxy, whose setup.py includes this:
```
install_requires=['requests>=1.1.0'],
```